proto/kava/bep3/v1beta1/bep3.proto

```diff
@@ -0,0 +1,113 @@
+syntax = "proto3";
+package kava.bep3.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/kava-labs/kava/x/bep3/types";
+
+// Params defines the parameters for the bep3 module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+
+  repeated AssetParam asset_params = 1 [(gogoproto.nullable) = false];
+}
+
+// AssetParam defines parameters for each bep3 asset.
+message AssetParam {
+  option (gogoproto.goproto_stringer) = false;
+
+  string      denom          = 1;
+  int64       coin_id        = 2 [(gogoproto.customname) = "CoinID"];
+  SupplyLimit supply_limit   = 3 [(gogoproto.nullable) = false];
+  bool        active         = 4;
+  string      deputy_address = 5 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  bytes fixed_fee = 6 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  bytes min_swap_amount = 7
+      [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  bytes max_swap_amount = 8
+      [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  uint64 min_block_lock = 9;
+  uint64 max_block_lock = 10;
+}
+
+// SupplyLimit define the absolute and time-based limits for an assets's supply.
+message SupplyLimit {
+  option (gogoproto.goproto_stringer) = false;
+
+  bytes limit = 1 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  bool  time_limited                        = 2;
+  google.protobuf.Duration time_period      = 3 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+  bytes                    time_based_limit = 4
+      [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+}
+
+// SwapStatus is the status of an AtomicSwap
+enum SwapStatus {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // AtomicSwap is unspecified
+  SWAP_STATUS_UNSPECIFIED = 0x00;
```

Comment:
is this the same as `SWAP_STATUS_UNSPECIFIED = 0`?

Reply:
Yep I think I copied it from the old source -- will remove the hex `0x` prefix, doesn't seem necessary and makes it less clear.